test/test_files/copy/copy_node.test

```diff
@@ -0,0 +1,48 @@
+-NAME Subset
```

Comment:
I think the convention we have in .test files is to end the test names with *Test. So Subset -> SubsetTest. CheckLines -> CheckLinesTest etc.